be/src/formats/parquet/statistics_helper.cpp

```diff
@@ -258,12 +258,12 @@ Status StatisticsHelper::in_filter_on_min_max_stat(const std::vector<std::string
         ColumnPredicate* pred_ge = pool.add(new_column_ge_predicate(get_type_info(ltype), 0, min_value));
         RETURN_IF_ERROR(pred_ge->evaluate_and(values.get(), filter.data()));
         if (!SIMD::contain_nonzero(filter)) {
-            selected[i] = false;
+            selected[i] = 0;
             continue;
         }
         ColumnPredicate* pred_le = pool.add(new_column_le_predicate(get_type_info(ltype), 0, max_value));
         RETURN_IF_ERROR(pred_le->evaluate_and(values.get(), filter.data()));
-        selected[i] = SIMD::contain_nonzero(filter);
+        selected[i] = SIMD::contain_nonzero(filter) ? selected[i] : 0;
```

Comment:
If selected[i]=0, is it necessary to calculate count_nonzero?

Reply:
done